Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1419,4 +1427,78 @@ - (void)serverCallback:(BNCServerResponse *)response {
     }
 }
 
+#pragma mark - Debugging functions
+
+- (void)bnc_addDebugGestureRecognizer {
+    [self bnc_addGesterRecognizer:@selector(bnc_connectToDebug:)];
+}
+
+- (void)bnc_addCancelDebugGestureRecognizer {
+    [self bnc_addGesterRecognizer:@selector(bnc_endDebug:)];
+}
+
+- (void)bnc_addGesterRecognizer:(SEL)action {
+    BNCLongPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:action];
```

Comment:
Maybe Qinwei can chime in here, but I believe he found this to be used by Mixpanel for their debugging option